src/components/dialogs/edit-package-details-dialog.tsx

```diff
@@ -81,11 +112,54 @@ export const EditPackageDetailsDialog = ({
         package_id: packageId,
         description: description,
         website: website,
+        license: license,
       })
       if (response.status !== 200) {
         console.error("Failed to update package details:", response.data)
         throw new Error("Failed to update package details")
       }
+      // get package files list
+      const packageFiles = []
+      const packageFilesResponse = await axios.post("/package_files/list", {
```

Comment:
Maybe use hooks where it can simplify the code

Reply:
yep there already exists a hook for this IIRC